docs/Variables.adoc

```diff
@@ -30,13 +33,21 @@ Required variable, but may be set to `none` where not applicable.
 | Name
 | Description
 
+| `pass:[__meta__]`
+| Metadata used by the Babylon infrastructure when running AgnosticD.
+This variable should be ignored by workloads, configs, and roles.
+
 | `agonsticd_callback_token`
 | Authentication Bearer token used to callback with information and data collected from `agnosticd_user_info`.
 This is used to integrate AgnosticD with external systems including Babylon and GUID Grabber.
-Default value `__meta__.callback.token`.
+Default value `pass:[__meta__.callback.token]`.
```

Comment:
What happen when `__meta__` (and thus `__meta__.callback.token`) is not defined?
=> Callback is ignored all together
I think we should make it clear

Reply:
I want to remove the use of `__meta__` in this case and have the babylon_anarchy_governor just set `agnosticd_callback_url` and `agnosticd_callback_token`.